google/cloud/storage/benchmarks/throughput_options.cc

```diff
@@ -177,6 +178,18 @@ google::cloud::StatusOr<ThroughputOptions> ParseThroughputOptions(
        [&options](std::string const& val) {
          options.download_stall_timeout = ParseDuration(val);
        }},
+      {"--minimum-sample-delay",
+       "configure the minimum time between samples."
+       " Sometimes we only want to collect a few samples per second."
+       " This can make the data resulting from a multi-day run more"
+       " manageable.",
+       [&options](std::string const& val) {
+         absl::Duration d;
+         if (absl::ParseDuration(val, &d)) {
+           options.minimum_sample_delay = std::chrono::milliseconds(-1);
+         }
```

Comment:
I presume there should be an `else` there (and that the arms of the conditional would then need to be reversed).

Reply:
Fixed.